client/src/components/ViewDocument.js

```diff
@@ -0,0 +1,84 @@
+import React, { Component } from "react";
+
+import { Header, Container, Table } from "semantic-ui-react";
+
+export class ViewDocument extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      document: "",
+      documentName: "",
+      submissionDate: "",
+      dueDate: "",
+      fromLanguageName: "",
+      toLanguageName: "",
+      status: "",
+      ownerName: ""
+    };
+  }
+
+  componentDidMount() {
+    const { id } = this.props.match.params;
+
+    fetch(`http://localhost:4000/api/documents/${id}`, {
```

Comment:
Remove localhost url here, otherwise it will not work on Heroku

Reply:
It is removed